### PR TITLE
Private Events

### PR DESCRIPTION
**Pull request by Davidosky007**
## Private Events
In this project 
* Association was used to link the different database tables together
*  simple forms gem was used to generate forms
* navigation links were added for easy movement across the view pages

### CODE REVIEW

File: app/helpers/events_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module EventsHelper
+  def event_form
+    rubocop :disable
```
Comment (umairarshadbutt) [inline app/helpers/events_helper.rb]: Please remove it

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Davidosky007 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes  👇.
- [ ] Add  unit tests for models validations and associations ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/unit_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))
- [ ] Add integration tests  for authentication and event management  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/integration_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))
- [x] View Demo in the README file redirecting to the repo, please remove it or update the live link
- [x] The path like `localhost:3000/events/new `should not be open without a login
- [x] Remove the `20210125174324_add_event_date_to_events.rb `from migration
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Changes Required! :recycle:
### Hi @Davidosky007 :clap: :clap:
#### Good job so far :+1: you have done well implementing the required changes. However, some of the required changes are missing/not implemented correctly, check my comments below :point_down:

#### DEMO LINK:
 - As mentioned in the previous review the demo link is still missing.

#### 2) TESTS:
 - As mentioned in the last review "Add unit tests for models associations" and integration tests for event management

#### Reviewed by Temitayo Adeleke

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the development branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Pull request merged**